x/schema/keeper/schema.go

```diff
@@ -17,11 +16,11 @@ import (
 )
 
 var (
-	url        = "https://api.ipfs.sonr.ws"
```

Comment:
Did you mean to commit this?

Reply:
yes, for now its safer to statically define the route rather than going through the cli env variables. this might fail when deployed as i found the env variables where not loading properly.